webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -22,4 +22,51 @@ const servicesComponent = require('./services.component.ejs');
 $("#sidebar-menu--cluster-view").addClass("active");
 $("#sidebar-menu--cluster-view--services").addClass("active");
 
-$('#content-wrapper').html("<div>TODO: List services on each node here.</div>");
\ No newline at end of file
+const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
+const loadingComponent = require('../../job/loading/loading.component.ejs');
+const serviceTableComponent = require('./service-table.component.ejs');
+const serviceViewComponent = require('./services.component.ejs');
+const loading = require('../../job/loading/loading.component');
+//const webportalConfig = require('../../../config/webportal.config.json');
+const userAuth = require('../../user/user-auth/user-auth.component');
```

Comment:
move `require` block to the beginning of js file

Reply:
Solved